Redux patterns/redux+typescript.md

```diff
@@ -3,19 +3,19 @@
 # Redux + Typescript - Writing good/quality code could actually mean less boilerplate 
 # Redux + Typescript - Writing quality Javascript was never so fast 
 # Redux + Typescript - Writing quality Javascript just became way faster
-# Redux + Typescript + Immer - Writing safe maintainble code just became blazing fast 
+# Redux + Typescript + Immer - Writing safe maintainable code just became blazing fast 
 # Redux patterns - Writing safe maintainable code just became blazing fast 
 ## Part 1 - Redux + Typescript 
 
 
-Redux is rightly known as the gold standard for managing application state in Javascript. It separates app state from ui elements, and organizes the most important part of an application - app state and app logic in a coherent way. But many programmers complain that Redux requires too much boilerplate/ is too verbose, and is just not worth it. Many avoid it, some use Mobx instead. 
+Redux is currently recognized as the gold standard for managing application state in ReactJS. It separates app state from UI elements, and organizes the most important parts of an application which are the app state and app logic in a coherent way. However, many programmers still complain that Redux requires too much boilerplate and is too verbose and therefore is just not worth it. Many avoid it entirely and some use Mobx instead. 
```

Comment:
rightly known vs currently recognized.

What you wrote is grammatically correct of course, I just have a feeling it might be too imposing right at the beginning of the article. That's just my general sentiment. It's up to you if you want this change.

Reply:
managing application state in `Javascript vs ReactJS`

This is to avoid people who might nitpick and is just a recommendation. I think both statements are probably true, although I'm not sure if it is for JavaScript since I don't think Facebook even uses it.

Reply:
While Redux is mostly identified with React, Redux is also popular for managing state in Angular. I think I also heard it is used for Vue.

Reply:
Also not sure about the title of the series - "just became blazing fast" - most of the methods in the series could be used for a while. The newest (use-immer) could be used for a year or so.

Reply:
You make a good point, ngrx and VueX are both used so I'll change it back to JavaScript. 

In terms of the title, I like it because I think it's a good hook to entice the user into reading more.